models/human.rb

```diff
@@ -17,13 +17,34 @@ def legal_name
     # @name.split(' ', 2).reverse.join(', ')
   end
 
-  # attr_accessor :job_title
-  def job_title=(value)
-    @job_title = value
-  end
+  attr_accessor :job_title
+  #def job_title=(value)
+#    @job_title = value
+#  end
+
+  #def job_title
+  #  @job_title
+#  end
+
+attr_accessor :age
+#  def age
+#    @age
+#  end
+
+#  def age=(value)
+#    @age = value
+#    end
+
+def to_drink
+  @to_drink
+end
 
-  def job_title
-    @job_title
-  end
+def to_drink
```

Comment:
I rename all to to_drink? and it gets annoyed with to_drink = true (unexpected = )